learnosity_sdk/request/dataapi.py

```diff
@@ -122,4 +129,11 @@ def request_iter(self, endpoint, security_packet,
             else:
                 yield data
 
+    @staticmethod
+    def __iteritems(obj, **kwargs):
```

Comment:
Would be good to get some documentation comments for this method so it's more clear what it does/what it's for.